lib/Sema/TypeCheckType.cpp

```diff
@@ -1533,6 +1548,9 @@ static Type resolveNestedIdentTypeComponent(TypeResolution resolution,
     memberTypes = TypeChecker::lookupMemberType(
         DC, parentTy, comp->getNameRef(), lookupOptions);
 
+  // Prefer Swift types over Clang types.
+  memberTypes = preferSwiftTypes(memberTypes);
```

Comment:
Can you add a test case? Should this go inside lookupMemberType() instead, since there are other callers of lookupMemberType() elsewhere?

Reply:
Will add a test case. I'm not sure about adding this to `lookupMemberType()` because code completion may show less results.